site/LAYOUT.md

```diff
@@ -84,31 +84,41 @@ It allows programmatically determining which versions of Jenkins get served whic
 
 ## Regular tiered update sites (LTS and weekly)
 
-Directories with names containing Jenkins LTS baselines like `2.204` are version specific update sites.
+The Jenkins update center provides multiple update sites.
+Each of them only offer plugins compatible with the respective baseline, as well as the newest weekly or LTS release, depending on the variant (LTS or weekly).
+This increases the usefulness of the update sites to users of slightly older releases of Jenkins, as they won't regularly be offered incompatible releases of plugins.
 
-* `2.204/` contains the tiered update site for Jenkins 2.204 and _similar_ weekly releases not matching an LTS baseline.
-* `stable-2.204/` contains the tiered update site for Jenkins 2.204.x.
+---
+**NOTE**:  The update center used to support formats like `2.204/` and `stable-2.204/`.
+This is no longer the case for recent versions.
 
-Each of these update sites only offer plugins compatible with the respective baseline, as well as the newest weekly or LTS release, depending on the variant (LTS or weekly).
-This increases the usefulness of the update sites to users of slightly older releases of Jenkins, as they won't regularly be offered incompatible releases of plugins.
+---
 
 The Jenkins project publishes a limited, fixed number of tiered update sites to encourage users to regularly update Jenkins. 
```

Comment:
The number is dynamic, what's fixed (now) is how old the oldest supported release can be.